docs/contributing/tasks.md

```diff
@@ -54,7 +54,7 @@ This task will:
 **`gulp test`**
 
 This task will:
-- Run scss:lint
+- Run scss:stylelint
```

Comment:
Given we no longer need to run both tasks side-by-side, can we rename the stylelint task to just 'lint', effectively preserving the existing take name and just updating it to use stylelint?